src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml.cs

```diff
@@ -225,16 +228,22 @@ private void VmOnWorkspacePropertyEditRequested(WorkspaceModel workspace)
 
```

Comment:
Since the view hierarchy is changed, the selection coordinates need to be translated before they can be used.